recipes/rapmap/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler:	
```

Comment:
@rob-p we did the switch everything will be compiled against GCC7 from now on.

Reply:
Woot!  Thanks for the heads up @bgruening .

Reply:
Your welcome!